tests/test_queries.py

```diff
@@ -910,13 +912,14 @@ def test_petrol(client: FlaskClient) -> None:
 
 
 def test_pic(client: FlaskClient) -> None:
-    """Pic module"""
+    """Pic module."""
 
     if not has_google_api_key():
         # NB: No Google API key on test server
         return
 
-    json = qmcall(client, {"q": "Sýndu mér mynd af Katrínu Jakobsdóttur"}, "Picture")
+    # TODO: Re-add test with "Katrín Jakobsdóttir" when fixed GreynirPackage is released
```

Comment:
Do we want to handle this ambiguity in GreynirPackage though? I think we should somehow try to infer the correct root of the name by popularity in the Greynir scraper database.

My idea, roughly, would be to create a function which lists all possible names (in nf.) matching the provided form, search our database for matching mentions of the name, and return the most popular one.
The function could then be used in this case (when showing an image of a person) and also in builtin.py (when asking about someone), or even in the unnecessary key from the opinion module.
(This would have a bit of overhead though, maybe we can optimize this in some way.)